### DISCUSSION TITLE
subdomain routing without conflicts

### DISCUSSION THREAD

**Discussion by pie6k** [vercel/next.js#10979] [Help]
Let's say my app has `team` entity, `post` entity and `category`

Possible routes

`https://team1.app.com/post1`
`https://team2.app.com/post2`

but also `https://app.com/cateogory1/post3` where `post3` is public in my app so can be accessed without the team.

I understand subdomain parsing is not part of next.js core for security reasons, but in example above first segment after `app.com` can mean either post or category, depending on if subdomain is used or not.

Is it possible to handle such case in next?

**Comment by timneutkens** [comment]
Moved this into discussions, not sure how you bypassed the issue template.

---

Yes this is possible using Next.js, you'd basically just create a catch-all page `pages/[...slug].js` or similar, and then you have to handle those cases in one page.

However in general I would recommend using a url structure that that allows for better splitting of pages.

**Reply by flutterforsale** [reply]
Hey @timneutkens 

Can i using SSR with subdomain page? 
My app is one page same architecture static.fun https://github.com/zeit/static-fun

Page of user: team1.static.fun, team2.static.fun 

I am looking a solution allow user can re-deploy (export static or SSG) after his update page data. 

Thanks!

**Reply by flutterforsale** [reply]
I am also using willcard domain success https://zeit.co/blog/wildcard-domains
